source/helpers/ValidationHelper.js

```diff
@@ -45,8 +45,13 @@ export const validateInput = (value, rules) =>
       /**
        * Validator only accepts strings.
        */
-      const valueToValidate = String(value);
-
+      let valueToValidate = String(value);
+      /**
+       * If the value comes from a checkbox or select input, we want the following values to count as empty, so that the isEmpty rule can be applied correctly.
```

Comment:
Very small thing but would be good to break the comment after "empty," so that we don't have to scroll right in order to se the end of the sentence 👍

Reply:
Fixed.